### PR TITLE
remove the dependency of minikube and hostname setting

### LINKED ISSUE #407: Need to add instructions to get IP for platforms other than minikube
Description of the issue: 

- In the [Deploy and Update] (https://github.com/oam-dev/rudr/blob/master/docs/tutorials/deploy_and_update.md) the document assumes Minikube is being used when getting the IP. For customers using K8's customers elsewhere it would be good to have the instructions 

- The exact location is in the [Visit the Web App](https://github.com/oam-dev/rudr/blob/master/docs/tutorials/deploy_and_update.md#visit-the-web-app) section 

- One suggestion for a fix would be to add a line letting users know they can get the External IP by doing `kubectl get svc`

### LINKED ISSUE #420: Setting example.com hostname seems unnecessary for the hello-world example
This introduces more steps and doesn't bring much value.

### PR DESCRIPTION
**Pull request by wonderflow**
* Use `port-forward` to get the url. fixes #407
* Remove the hostname setting. fixes #420

### CODE REVIEW

File: docs/tutorials/deploy_and_update.md [hunk 1]
```diff
@@ -236,23 +236,22 @@ status:
...
-   ```shell script
-   echo "192.168.99.101 example.com" >> /etc/hosts
-   ```
-3. Then you can directly find what you want by `curl`
-    ```shell script
-    $ curl example.com
-    Hello Rudr!
-    ```
+
+Let's use `port-forward` to help us, get the application URL by running these commands:
```
Comment (suhuruli) [inline docs/tutorials/deploy_and_update.md]:
```suggestion
Let's use `port-forward` to help us get the application URL by running these commands:
```

File: docs/tutorials/deploy_and_update.md [hunk 2]
```diff
@@ -236,23 +236,22 @@ status:
...
+
+```
+export POD_NAME=$(kubectl get pods -l "oam.dev/instance-name=first-app-helloworld-python-v1,app.kubernetes.io/name=first-app" -o jsonpath="{.items[0].metadata.name}")
+echo "Visit http://127.0.0.1:9999 to use your application"
+kubectl port-forward $POD_NAME 9999:9999
+```
+
+`kubectl port-forward` command will block and serving your requests, you could open your browser to visit this URL.
```
Comment (suhuruli) [inline docs/tutorials/deploy_and_update.md]: Not following this sentence.

File: docs/tutorials/deploy_and_update.md [hunk 3]
```diff
@@ -349,20 +348,21 @@ status:
...
+Let's visit the web app again and find the following result:
 
-```console
-$ curl oamexample.com
+```
 Hello World!
 ```
 
-The response from the url indicates our change of environment has successfully went into effect.
+The response indicates our change of environment has successfully went into effect.
```
Comment (suhuruli) [inline docs/tutorials/deploy_and_update.md]:
```suggestion
The response indicates our change of environment is successful. 
```

File: docs/tutorials/deploy_and_update.md [hunk 4]
```diff
@@ -236,23 +236,22 @@ status:
...
+
+Let's use `port-forward` to help us get the application URL by running these commands:
+
+```
+export POD_NAME=$(kubectl get pods -l "oam.dev/instance-name=first-app-helloworld-python-v1,app.kubernetes.io/name=first-app" -o jsonpath="{.items[0].metadata.name}")
+echo "Visit http://127.0.0.1:9999 to use your application"
+kubectl port-forward $POD_NAME 9999:9999
+```
+
+`kubectl port-forward` command will block and serving your requests.
```
Comment (suhuruli) [inline docs/tutorials/deploy_and_update.md]:
```suggestion
`kubectl port-forward` command will block and serve your requests.
```

**Comment by suhuruli** [comment]
LGTM after one small nit. Feel free to merge tonight (working day in China).

**Comment by wonderflow** [comment]
@suhuruli  thanks!

**Pull request merged**